docs/source/config-file.rst

```diff
@@ -61,6 +61,15 @@ The config file is made up of multiple parts
 * Here you can configure multiple JIRA instances if you have projects with differing downstream JIRA instances.
   Ensure to name them appropriately, in name of the JIRA instance above is `example`.
 
+.. code-block:: python
+
+    "default_jira_fields": {
+      "storypoints": "customfield_12310243"
+    },
+
+* Specify the jira field for syncing. As of now the supported field is `storypoints`. It is also possible to
+  add `priority`, but in the abscence the default value will be `priority`.
```

Comment:
"abscence" ==> "absence"

I might insert "only" before "supported", but I think the third sentence could be clearer.  I would be inclined to rework this whole description.  I think we ought to describe this mapping in terms of `github_project_fields`.  That is, the keys in `default_jira_fields` correspond to the values available to be specified in `github_project_fields`, and the keys' values provide the names of the upstream fields which are used to fetch the values for those operations.

Currently, we support only `"storypoints"` and `"priority"`.  By default, the value for the priority is sourced from the upstream `"priority"` field, but providing a `"priority"` key for `default_jira_fields` will override that.  Unfortunately, there is no default available for sourcing story points, so, the `"storypoints"` key must be provided if projects wish to sync story point values, and, for our corporate Jira instance, the value should be specified as `"customfield_12310243"`.

Reply:
Done. e4272df08a24e90b08afac9e97bb198e7baa37bc